File: Ass_12/Assignment_3.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i= 0;
    int j = 0;
    
    for(i = 1; i <= iRow; i++)
    {
        for(j = iCol; j >= 1; j--)
        {
            printf("%d\t",j);
        }
        printf("\n");
    }
   

}
int main()
{
    int iValue1 = 0 , iValue2 = 0;

    printf("Enter number of rows:\n");
    scanf("%d",&iValue1);

    printf("Enter number of columns:\n");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;



}

File: Ass_50/Assignment_3.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;

        for(i = 0; i < iRow; i++)
        {
            for(j = 0; j < iCol; j++)
            {
                if(i == 0 || i == iRow - 1 || j == 0 || j == iCol - 1 || (i + j == iCol - 1))
                {
                    System.out.print(" * ");
                }
                else
                {
                    System.out.print("   ");
                }
            }
            System.out.println();
        }
    }
}

class Assignment_3
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter the number of columns :");
        int iCol = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow, iCol);
    }
}
